src/shared/install.h

```diff
@@ -78,6 +79,13 @@ enum UnitFileChangeType {
         _UNIT_FILE_CHANGE_INVALID = INT_MIN
 };
 
+enum UnitFileFlags {
+        UNIT_FILE_RUNTIME = 1,
+        UNIT_FILE_FORCE = 1 << 1,
+        _UNIT_FILE_FLAGS_MAX,
+        _UNIT_FILE_FLAGS_INVALID = -1
```

Comment:
I don't remember seeing these before... The "new" github behavior around force pushes feels weird sometimes. Anyway, will fix.